lib/galaxy/config.py

```diff
@@ -305,7 +305,7 @@ def __init__( self, **kwargs ):
             self.use_tool_dependencies = True
         else:
             self.tool_dependency_dir = None
-            self.use_tool_dependencies = False
+            self.use_tool_dependencies = os.path.exists(self.dependency_resolvers_config_file)
```

Comment:
```
         if 'tool_dependency_dir' in kwargs:
             self.tool_dependency_dir = resolve_path( kwargs.get( "tool_dependency_dir" ), self.root )
             # Setting the following flag to true will ultimately cause tool dependencies
             # to be located in the shell environment and used by the job that is executing
             # the tool.
             self.use_tool_dependencies = True
         else:
             self.tool_dependency_dir = None
-            self.use_tool_dependencies = False
+            self.use_tool_dependencies = os.path.exists(self.dependency_resolvers_config_file)
```

@nsoranzo This is all in the ``tool_dependency_dir`` not set ``else`` block. If ``tool_dependency_dir`` is set ``use_tool_dependencies`` will always be ``True`` still.